spring-cloud-gcp-autoconfigure/src/test/java/com/google/cloud/spring/autoconfigure/secretmanager/SecretManagerConfigDataLoaderUnitTests.java

```diff
@@ -21,18 +26,30 @@ class SecretManagerConfigDataLoaderUnitTests {
   private final ConfigDataLoaderContext loaderContext = mock(ConfigDataLoaderContext.class);
   private final GcpProjectIdProvider idProvider = mock(GcpProjectIdProvider.class);
   private final SecretManagerTemplate template = mock(SecretManagerTemplate.class);
+  private final GcpSecretManagerProperties properties = mock(GcpSecretManagerProperties.class);
   private final ConfigurableBootstrapContext bootstrapContext = mock(
       ConfigurableBootstrapContext.class);
   private final SecretManagerConfigDataLoader loader = new SecretManagerConfigDataLoader();
 
-  @Test
-  void loadIncorrectResourceThrowsException() {
+  @ParameterizedTest
+  @CsvSource({
+      "regional-fake, us-central1",
+      "fake, "
+  })
+  void loadIncorrectResourceThrowsException(String resourceName, String location) {
     when(loaderContext.getBootstrapContext()).thenReturn(bootstrapContext);
     when(bootstrapContext.get(GcpProjectIdProvider.class)).thenReturn(idProvider);
     when(bootstrapContext.get(SecretManagerTemplate.class)).thenReturn(template);
+    when(bootstrapContext.get(GcpSecretManagerProperties.class)).thenReturn(properties);
     when(template.secretExists(anyString(), anyString())).thenReturn(false);
+    when(properties.getLocation()).thenReturn(Optional.ofNullable(location));
     SecretManagerConfigDataResource resource = new SecretManagerConfigDataResource(
-        ConfigDataLocation.of("fake"));
-    assertThatCode(() -> loader.load(loaderContext, resource)).doesNotThrowAnyException();
+        ConfigDataLocation.of(resourceName));
+    assertThatCode(() -> {
+      ConfigData configData = loader.load(loaderContext, resource);
+      SecretManagerPropertySource propertySource =
+          (SecretManagerPropertySource) configData.getPropertySources().get(0);
+      assertThat(Optional.ofNullable(location)).isEqualTo(propertySource.getLocation());
```

Comment:
First, I'm not from Google, so my opinion is truly not more than my opinion. Google may see things different 😉 

Now, in assertions you normally test assumtions/expectations on an object. Having this on mind the assertions should be more in this way:
```text
assertThat(objectToTest)
  .verificationOne(...)
  .verificationTwo(...);
```

The result is the same, but now it is in a more logical order:

```java
assertThat(propertySource)
  .returns(Optional.ofNullable(location), SecretManagerPropertySource::getLocation);
```
You could add more verifications if suitable.

Reply:
I followed the existing convention used throughout the codebase, where assertions are written in the same order as the current pattern. However, I understand your perspective, and if the team prefers the alternative style you suggested, I’m happy to make the adjustment.